misc/bootstrap.sh

```diff
@@ -7,6 +7,7 @@ touch /var/log/cron/owncloud.log
 test -e /owncloud/config.php || cp /root/owncloud_config.php /owncloud/docker_image.config.php
 test -e /owncloud/docker_image_owncloud.config.php || cp /root/docker_image_owncloud.config.php /owncloud/docker_image_owncloud.config.php
 test -e /owncloud/3party_apps.conf || cp /root/3party_apps.conf /owncloud/
+test -e /owncloud/dhparam.pem || openssl dhparam -out /owncloud/dhparam.pem 4096
```

Comment:
Yes you're right, just fixed it.

2016-02-10 0:04 GMT+01:00 Robin Schneider <notifications@github.com>:

> In misc/bootstrap.sh
> <https://github.com/jchaney/owncloud/pull/43#discussion_r52389649>:
>
> > @@ -7,6 +7,7 @@ touch /var/log/cron/owncloud.log
> >  test -e /owncloud/config.php || cp /root/owncloud_config.php /owncloud/docker_image.config.php
> >  test -e /owncloud/docker_image_owncloud.config.php || cp /root/docker_image_owncloud.config.php /owncloud/docker_image_owncloud.config.php
> >  test -e /owncloud/3party_apps.conf || cp /root/3party_apps.conf /owncloud/
> > +test -e /owncloud/dhparam.pem || openssl dhparam -out /owncloud/dhparam.pem 4096
>
> Can you move this to the if branch which handles HTTPS: echo "Copying
> nginx.conf with SSL support …"? There is no need to generate the file
> when it is not used anyway. I for example use a reverse proxy which
> terminates TLS outside of this container.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jchaney/owncloud/pull/43/files#r52389649>.
>